wolfssl/wolfcrypt/types.h

```diff
@@ -113,22 +113,27 @@ decouple library dependencies with standard string, memory and so on.
 
     #if defined(_MSC_VER) || defined(__BCPLUSPLUS__)
         #define WORD64_AVAILABLE
+        #undef W64LIT
```

Comment:
It's a very specific macro conflict (not common). Did you suggest to the customer to undef it the same way in their user_settings.h? Otherwise doing a single #undef at the top of even an `#ifndef W64LIT` might be better.